src/api/validators/index.ts

```diff
@@ -0,0 +1,18 @@
+import { getValidatorPerformance, ValidatorPerformance } from './validators';
+
+const TIMEOUT = 5 * 60 * 1000;
+
+export const validatorPerformance = async ctx => {
+  try {
+    ctx.request.socket.setTimeout(TIMEOUT);
+    const validatorPerformance: ValidatorPerformance | undefined = await getValidatorPerformance();
```

Comment:
since `getValidatorPerformance` is not an async function, there's no need to set a socket timeout nor to await the `getValidatorPerformance` call